### ISSUE TITLE
Server Call In Loss Function During Training

### ISSUE THREAD

**Issue by crvogt** [keras-team/keras#12816]
I haven't been able to find anything on this (and I'm pretty sure I know why). 

- I'm looking to send the output from a network to a server at the start of a custom loss function call. 
- The server would then send the data back, modified by a commercial software. - The loss would then be calculated based on the data received from the server.

I'm sure I'm not the first to think of this, but is it even reasonable at the moment?
